### PR TITLE
Added support for appending kernel parameters during installation

### PR DESCRIPTION
**Pull request by abdhaleegit**
Provide user an option to give kernel parameter option to be enabled while OS installation

Signed-off-by: Abdul Haleem <abdhalee@linux.vnet.ibm.com>

### CODE REVIEW

File: installvm.py [hunk 1]
```diff
@@ -437,6 +440,10 @@ def createKickstart(self):
 
         partition_string = ''
         multipath_string = ''
+        kernel_params = 'mitigations=auto quiet crashkernel=1024M '
+        if vmParser.args.kernel_params != '':
+            kernel_params=kernel_params+vmParser.args.kernel_params
```
Comment (PraveenPenguin) [inline installvm.py:445]: please use string way to concatenate string, i know we have other places as well if possible can you please send another fix commit for same

File: lib/configparser.py [hunk 2]
```diff
@@ -51,6 +51,8 @@ def parse_args(self, argv=None):
             '--boot-disk', help='boot disk ID from VIOS to set order ex: U9080.M9S.78264B8-V1-C101-T1-L8100000000000000', required=False)
         hostGroup.add_argument(
             '--multipathsetup', help='Host disk having multipath setup', default='')
+        hostGroup.add_argument(
+            '--kernel-params', help='append addon kernel parameters', default='')
```
Comment (PraveenPenguin) [inline lib/configparser.py:55]: cool , please update ReadME as well

Comment (abdhaleegit) [reply lib/configparser.py:55]: done

File: installvm.py (hunk 1 above)
Comment (abdhaleegit) [reply installvm.py]: changed above, the whole code cleanup can be different commit

**Review by PraveenPenguin — APPROVED** [review]

**Pull request merged**